install_night.html

```diff
@@ -36,15 +36,43 @@
       <!-- Any section element inside of this container is displayed as a slide -->
       <div class="slides">
         <!-- Opening slide -->
+        <section data-background-iframe="https://developer.apple.com/xcode/downloads/">
+          <h2>Installing Xcode</h2>
+          <p>Head on over to: https://developer.apple.com/xcode/downloads/</p>
+          <img style="max-width:370px; min-width:330px;" src="https://devimages.apple.com.edgekey.net/xcode/images/downloads_2x.png">
+        </section>
         <section>
-          <h3>Welcome!</h3>
-          <p>Jeff, this is you buddy :) Setup Instructions for Install Night go here.</p>
+          <h2>CocoaPods</h2>
+          <ul>
+              <li>Manages library dependencies for Xcode projects</li>
+              <li>Dependencies are specified in a single text file called a Podfile</li>
+              <li>Resolves dependencies between libraries, fetches the required source code and links everything together in an Xcode workspace to build your project</li>
+          </ul>
+          <p>Reference: </p> <a href="https://guides.cocoapods.org/using/getting-started.html">https://guides.cocoapods.org/using/getting-started.html</a>
+
         </section>
         <section>
-          <p>Second Slide</p>
+          <h2>Installing CocoaPods</h2>
+          <ol>
+              <li>Open Terminal and enter:</li>
+              <blockquote style="max-width:700px; min-width:700px;">sudo gem install cocoapods</blockquote>
```

Comment:
If you like, you can replace `<blockquote></blockquote>` with `<pre><code>sudo gem install cocoapods</code></pre>` to add clarity that the command doesn't require quotation marks. Looks something like this: 

![code_example](https://cloud.githubusercontent.com/assets/4016985/7671238/f99ec4cc-fc93-11e4-82d2-7bb08c786a50.png)